returnn/torch/engine.py

```diff
@@ -105,6 +113,11 @@ def init_train_from_config(
         """
         assert config is self.config
         super().init_train_from_config(config=config)
+        if self._use_DDP:
+            ddp_local_rank = int(os.environ["LOCAL_RANK"])
```

Comment:
You should not use the `os.environ` but rather the `torch.distributed` functions.